### PR TITLE
Fix typo README

### PR DESCRIPTION
**Pull request by Olexandr88**
Update link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Search for bugs:
 
 SWT used to track ongoing development and issues in Bugzilla .
 
-- <https://bugs.eclipse.org/bugs/buglist.cgi?product=Platform&component=SWT>
+- <https://bugs.eclipse.org/bugs/buglist.cgi?component=SWT&product=Platform>
```
Comment (laeubi) [inline README.md:116]: We are now using Github issues...

Comment (Wittmaxi) [inline README.md:116]: Both links lead to the same page, what does this change fix?

Comment (Olexandr88) [reply README.md:116]: Yes, I completely agree with you, but in my opinion the second link is more correct

**Review by laeubi — CHANGES_REQUESTED** [review]
Both links are equivalent (and both are wrong), but none of them are "typo".

If you interested in changing the readme please

1. Change Bugzilla references to github reference
2. Include a useful description in the commit message

File: README.md (hunk 1 above)
Comment (Phillipus) [reply README.md:116]: Either the link should be removed, or the title and rubric changed to mention that BugZilla is no longer used and this is the link to old (archived) bugs.

File: README.md [hunk 2]
```diff
@@ -111,9 +111,9 @@ Contact the project developers via the project's "dev" list.
 Search for bugs:
 ----------------
 
-SWT used to track ongoing development and issues in Bugzilla .
+Github is used to track current issues
 
-- <https://bugs.eclipse.org/bugs/buglist.cgi?product=Platform&component=SWT>
+- <https://github.com/issues?user=eclipse-platform>
```
Comment (Phillipus) [inline README.md:116]: The URL for SWT should be `https://github.com/eclipse-platform/eclipse.platform.swt/issues`

But is there any point in this? This README file is on the front page of the GH with the "Issues" menu right at the top of the page.

Comment (Olexandr88) [reply README.md:116]: So I think it doesn't make sense. Then you can remove this section entirely, as one of the reviewers advised me to do

**Comment by merks** [comment]
At some point someone will ask you to squash your commits.  Amending a commit and force pushing it is typically better.  Here we have 3 commits which no one will want to merge as is.

**Pull request closed without merging**